src/index.html

```diff
@@ -1,13 +1,39 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
         content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <section class="card" data-qa="card">
+      <img
+       src="./images/imac.jpeg"
+       alt="image"
+       class="card__image"
+      >
+      <h1 class="card__title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h1>
+      <h2 class="card__subtitle">Product code: 195434</h2>
+      <div class="card__raiting-block">
+        <div class="stars stars--4">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+        <span class="card__rewiew">Reviews: 5</span>
+      </div>
+      <div class="card__price-block">
+        <span class="card__price-block-name">Price:</span>
+        <span class="card__price-block-value">$2,199</span>
+      </div>
+      <a href="#"
+       class="card__button"
+       data-qa="hover">Buy
+      </a>
```

Comment:
fix indentation (you lost one space before each attribute)
and move ">" to a new line